webpack/webpack.config.server.js

```diff
@@ -3,7 +3,13 @@ import webpack from 'webpack';
 import HappyPack from 'happypack';
 
 const root = process.cwd();
-const serverInclude = [path.join(root, 'src', 'server'), path.join(root, 'src', 'universal')];
+const serverInclude = [
+  path.join(root, 'src', 'server'),
+  path.join(root, 'src', 'universal')
+];
+const serverExclude = [
```

Comment:
glad you included this! another option is to keep migrations out of the `src`, same as we do with `build`, `webpack`, etc. 
Not sure which you think is cleaner.